CustomTextAnalytics.CognitiveSearch/Solution/CustomTextAzureFunction/CustomSkillsetFunction/Function.cs

```diff
@@ -43,49 +44,37 @@ public static class Function
             // get custom text app secrets from header
             var customtextEndpoint = request.Headers["endpointUrl"];
             var customtextKey = request.Headers["apiKey"];
-            var customtextProjectName = request.Headers["projectName"];
-            var customtextDeploymentName = request.Headers["deploymentName"];
+            var recognizeCustomEntitiesProjectName = request.Headers["projectName"];
```

Comment:
keep the 'Custom Text' naming as it is in the program.cs
you can use recognizeCustomEntitiesProjectName internally in the service

Reply:
each action will have its own project name and deployment name